src/h5web/toolbar/controls/ColorMapSelector/ColorMapSelector.tsx

```diff
@@ -1,25 +1,11 @@
-import {
-  CYCLICAL,
-  DIVERGING,
-  INTERPOLATORS,
-  RECOMMENDED,
-  MULTI_HUE,
-  SINGLE_HUE,
-} from '../../vis-packs/core/heatmap/interpolators';
-import type { ColorMap } from '../../vis-packs/core/heatmap/models';
-import { getLinearGradient } from '../../vis-packs/core/heatmap/utils';
-import Selector from './Selector/Selector';
+import { INTERPOLATORS } from '../../../vis-packs/core/heatmap/interpolators';
+import type { ColorMap } from '../../../vis-packs/core/heatmap/models';
+import { getLinearGradient } from '../../../vis-packs/core/heatmap/utils';
+import Selector from '../Selector/Selector';
 import styles from './ColorMapSelector.module.css';
-import ToggleBtn from './ToggleBtn';
+import ToggleBtn from '../ToggleBtn';
 import { FiShuffle } from 'react-icons/fi';
-
-const COLORMAP_GROUPS = {
-  Recommended: Object.keys(RECOMMENDED) as ColorMap[],
-  'Single hue': Object.keys(SINGLE_HUE) as ColorMap[],
-  'Multi hue': Object.keys(MULTI_HUE) as ColorMap[],
-  Cyclical: Object.keys(CYCLICAL) as ColorMap[],
-  Diverging: Object.keys(DIVERGING) as ColorMap[],
-};
```

Comment:
I hard-code the arrays and move `COLORMAP_GROUPS` in a separate file (see below). It duplicates the `ColorMap` values a bit , but the typing is less hacky and we do a lot less work on initial load since there are no longer any `Object.keys()`.